lib/active_merchant/billing/gateways/braintree_blue.rb

```diff
@@ -171,10 +171,6 @@ def unstore(customer_vault_id, options = {})
       end
       alias_method :delete, :unstore
 
-      def supports_network_tokenization?
```

Comment:
confirmed with Andre, `Braintree Blue` might not always support network tokenization so we'll need to verify it.